spec/coverage.rb

```diff
@@ -6,5 +6,5 @@
   formatters << Coveralls::SimpleCov::Formatter
 end
 
-SimpleCov.formatter = SimpleCov::Formatter::MultiFormatter[*formatters]
+SimpleCov.formatter = SimpleCov::Formatter::MultiFormatter.new(formatters)
```

Comment:
You're right - this change doesn't belong in here. We should sort this out in the master branch.